pkg/ddl/000027_commit_generation.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+    ALTER TABLE graveler_commits
+        ADD COLUMN generation INT;
+    WITH RECURSIVE cte AS
```

Comment:
How did we test this?

Reply:
Just manual testing. I understand the concern, will try to create a better test